### ISSUE TITLE
Request for Instructions on Building AstroDX from Source Code

### ISSUE THREAD

**Issue by Neuron-Group** [2394425147/astrodx#161]
Hello, I hope this message finds you well. I would like to politely request assistance regarding the method for building AstroDX from source code. I am interested in attempting to port AstroDX to Linux. I noticed that it seems to be a Unity project; can I build it through Unity?

**Comment by DavidScann** [comment]
Hey there! Although I am a Discord moderator for the official Discord server, which means I cannot directly provide you with assistance or access regarding game code, I can partially provide some information related to your issue.

AstroDX is an open-core/partial source available project: You are welcome to read and examine the code, but I **believe** under the current license you're not allowed to redistribute or use it personally or commercially. (I may be wrong)

We do have plans to eventually release a Desktop build, which will definitely include a native Linux build for the game. It's certainly possible, including controller support. We just need to get other priorities out of the way first; mostly bugfixing and feature updates for mobile.

For now, AstroDX via Waydroid seems to be borked, so I don't think there's any real way to run AstroDX on Linux. Maybe via QEMU, I haven't tried that yet personally, but that _would_ come with immense performance overhead.

Also, I just want to thank you for expressing interest and for the really polite and nice issue!

**Comment by Soneoylys** [comment]
I've tried to build to linux and can get my desktop & notebook working (fedora40), so hopefully after we release the desktop version, the linux version will also be compiled and available for test.

**Comment by trustytrojan** [comment]
Bit of a bummer the full game code is not public, but I'm sure you have your reasons (probably sega?). Hopefully you guys are making progress with the native Linux build (Windows would be a nice bonus and would increase the playerbase)!

**Comment by Soneoylys** [comment]
> Bit of a bummer the full game code is not public, but I'm sure you have your reasons (probably sega?). Hopefully you guys are making progress with the native Linux build (Windows would be a nice bonus and would increase the playerbase)!

I don't think we have enough tester and devices available to validate linux build, but windows build is planned.

**Comment by 2394425147** [comment]
Hi! Currently AstroDX contains multiple paid assets from Unity's asset store. There are plans to substitute them with open-source versions. A pre-built Linux version is something I want to look into so thanks for the issue!
